### PR TITLE
Use "limit" instead of "mode" for Numeric state trigger and condition

### PR DESCRIPTION
**Pull request by NoRi2909**
## Proposed change

Currently the labels for "Above" and "Below" sections of the Numeric state trigger and condition are called "Above mode" and "Below mode":

image

As modes are always exclusive – you can't have two modes enabled at the same time – this labeling is misleading as you can specify _both_ a lower and an upper limit.

By changing them to "Lower limit" (instead if "Above mode") and "Upper limit" (instead of "Below mode") it becomes clear that the user can specify either or both at the same time.

As the field labels remain "Above" and "Below" and the summary also explains it that way it remains clear the limits are exclusive.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by bramkragten** [comment]
The label is actually just for the radio, but I get your point

File: src/translations/en.json [hunk 1]
```diff
@@ -3166,8 +3166,8 @@
                   "label": "Numeric state",
                   "above": "Above",
                   "below": "Below",
-                  "mode_above": "Above mode",
-                  "mode_below": "Below mode",
+                  "mode_above": "Lower limit",
+                  "mode_below": "Upper limit",
```
Comment (bramkragten) [inline src/translations/en.json:3170]: I think we should rename the key names if we are going to use them as a description for the entire section.

Comment (NoRi2909) [reply src/translations/en.json:3170]: Sure makes sense for consistency. Then we need to also rename them for the condition as well as this references the labels from the trigger. Shall I go ahead with that?

Comment (bramkragten) [reply src/translations/en.json:3170]: Yes please 👍

Comment (NoRi2909) [reply src/translations/en.json]: I made the additional changes in the strings.json but I'm afraid that the necessary changes in

ha-automation-trigger-numeric_state.ts
ha-automation-condition-numeric_state.ts

are beyond my rudimentary  programming skills and what the GitHub web editor allows me to mess with …

**Review by bramkragten — APPROVED** [review]

File: src/translations/en.json (hunk 1 above)
Comment (NoRi2909) [reply src/translations/en.json]: All checks have passed, but the changes are quite extensive so I can't judge if that is going to break something else.

**Pull request merged**